packages/modules/itwin-error-handling-react/src/components/toaster/BaseToastStyling.scss

```diff
@@ -112,25 +112,25 @@
   width: 48px;
 }
 
-.bwc-toast-wrapper
-.bwc-toast-success
-.status-icon-container
-.status-icon-background
-.status-icon {
+.itwin-toast-wrapper
```

Comment:
minor comment, maybe we should add stylelint to this monorepo if its not already present?